zipkin-cassandra-core/src/main/java/org/twitter/zipkin/storage/cassandra/Repository.java

```diff
@@ -757,69 +732,6 @@ private String debugInsertTraceIdByAnnotation(ByteBuffer annotationKey, long tim
                 .replace(":ttl_", String.valueOf(ttl));
     }
 
-    public Map<Long,Long> getTraceDuration(boolean head, long[] traceIds) {
```

Comment:
@michaelsembwever fyi this no longer has a caller, so it and its supporting things are nuked.